Automattic/wp-calypso
# Jetpack pricing page: add WAF to Scan & Security cards

### Changes proposed in this Pull Request

This PR adds the WAF (Web Application Firewall) to the list of features in the Scan and Security products cards.

### Testing instructions
- Spin-off this PR and visit http://jetpack.cloud.localhost:3000/pricing, or use the live branch feature
- Check that Scan and Security mention the WAF

### Screenshots
<img width="200" alt="" src="https://user-images.githubusercontent.com/1620183/189977846-135de62a-3d3e-4918-97ff-9407bdc6e0ad.png"> <img width="200" alt="" src="https://user-images.githubusercontent.com/1620183/189977848-3a7a3482-2f70-4a92-bdac-757eb8c33018.png">
